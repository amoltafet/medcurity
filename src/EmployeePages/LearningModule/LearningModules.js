import React from "react";

/**
 * Returns a essentially blank page. Serves to handle the postSlug.
 * @returns 
 */
function LearningModules() {
  return (
    <div >
      <div >
        <h1 className="text-center mt-5">Custom Learning Module page?</h1>
      </div>
    </div>
  );
}

export default LearningModules;