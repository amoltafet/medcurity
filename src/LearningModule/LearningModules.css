.uvs-left {
    border-radius: 10px 10px 10px 10px;
}
      
.uvs-right {
    border-radius: 10px 10px 10px 10px;
    
}