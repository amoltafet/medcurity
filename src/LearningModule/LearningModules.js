import React from "react";
import { useState, useEffect } from "react";
import axios from 'axios';
import { Link } from "react-router-dom"

/**
 * Returns a essentially blank page. Serves to handle the postSlug.
 * @returns 
 */
function LearningModules() {
  const [content, setContent] = useState([])
  const string = 'LearningModules'

  useEffect(() => {
    axios.get('http://localhost:3002/api/getData', { params: { table: string } }).then((response) => {
          setContent(Object.values(response.data))
      });
  }, [])

  return (
    <div className="home">
      <div class="container">
        <h1 className="text-center mt-5">Custom Learning Module page???</h1>
        {content.map((val) => {
          return (<h2>
            Description: {val.Description}
          </h2>);
        })}
      </div>
    </div>
  );
}

export default LearningModules;