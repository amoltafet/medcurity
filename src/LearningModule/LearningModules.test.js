import renderer from 'react-test-renderer';
import LearningModules from './LearningModules';
import React from 'react';

test('renders LearningModules', () => {
  const tree = renderer.create(<LearningModules />).toJSON();
  expect(tree).toMatchSnapshot();
});