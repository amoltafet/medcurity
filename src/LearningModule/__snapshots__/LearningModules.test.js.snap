// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LearningModules 1`] = `
<div>
  <div>
    <h1
      className="text-center mt-5"
    >
      Empty Page. Go back
    </h1>
  </div>
</div>
`;
